### PR TITLE
Feature/marvin o

### PR DESCRIPTION
**Pull request by fhanrath**
Controlling npcs works.

But they still receive ai events.
E.g. when you get attacked as an npc, you automatically draw your weapon and cannot move until you put it away.

### CODE REVIEW

**Comment by fhanrath** [comment]
I will  squash the commits as soon as the branch is ready for merge.

**Comment by Try** [comment]
For ai-events you can just disable them:
```c++
  for(size_t i=0;i<PERC_Count;++i)
    setPerceptionDisable(PercType(i));
```

File: game/world/world.cpp [hunk 1]
```diff
@@ -117,6 +117,19 @@ void World::insertPlayer(std::unique_ptr<Npc> &&npc, std::string_view waypoint)
     game.script()->setInstanceNPC("HERO",*npcPlayer);
   }
 
+void World::setPlayer(Npc* npc) {
+  if(npc==nullptr)
+    return;
+  npcPlayer->setProcessPolicy(Npc::ProcessPolicy::AiNormal);
+  npcPlayer->aiPush(AiQueue::aiContinueRoutine());
```
Comment (Try) [inline game/world/world.cpp:124]: `aiPush` is deferred command, better to refactor this into immediate call (to avoid `clearAiQueue` followup)

**Comment by fhanrath** [comment]
Thank you!
I integrated the changes and will squash the commits now :)

**Pull request closed without merging**